packages/ove-app-audio/src/index.js

```diff
@@ -102,9 +108,9 @@ app.get('/operation/:name(' + operationsList + ')', function (req, res) {
             message.operation.volume = req.query.volume;
         }
         if (name === Constants.Operation.SETPOSITION) {
```

Comment:
Perhaps it would be better to have the constants named as `SET_POSITION` instead of `SETPOSITION` when they were more than one word. This convention is followed for some constants but not followed for some others.

Reply:
with resharper changed all to _'s

Reply:
By the way @senakafdo is there any way to make ESLint catch this? Resharper is hinting at it very nicely but ESLint doesn't see it at all...

Reply:
@davidbirchwork, ESLint doesn't have a spell-checker AFAIK, there are other linters that you can use in VS if you wanted to (same with codacy), but R# is perhaps the best for things like this. On my VS code I use some other tools to check things like this and spelling mistakes.